src/test/java/com/training/baseTest/FlipkartBaseTest.java

```diff
@@ -0,0 +1,34 @@
+package com.training.baseTest;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import java.util.concurrent.TimeUnit;
+
+public class FlipkartBaseTest {
+    public WebDriver driver;
+    @BeforeEach
```

Comment:
add spaces between a function/annotation and list of globally defined variables.